analytics_data_api/v0/middleware.py

```diff
@@ -1,25 +1,27 @@
+from __future__ import absolute_import
+
 import abc
+
+import six
 from django.http.response import JsonResponse
 from rest_framework import status
 
 from analytics_data_api.v0.exceptions import (
+    CannotCreateReportDownloadLinkError,
     CourseKeyMalformedError,
     CourseNotSpecifiedError,
     LearnerEngagementTimelineNotFoundError,
     LearnerNotFoundError,
     ParameterValueError,
-    ReportFileNotFoundError,
-    CannotCreateReportDownloadLinkError,
+    ReportFileNotFoundError
```

Comment:
Could you add `include_trailing_comma = True` to the isort configuration in setup.cfg and re-run it on at least this file?  Example: https://github.com/edx/cookiecutter-django-app/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/setup.cfg#L2 .  It can help reduce diff churn like this.